src/test/java/edu/kis/powp/jobs2d/Job2dDriverTest.java

```diff
@@ -15,7 +15,7 @@
 	 * Driver test.
 	 */
 	public static void main(String[] args) {
-		FiguresJoe.figureScript1(driver);
+		//FiguresJoe.figureScript1(driver);
```

Comment:
In revision repo it is safe to delete code instead of comment. Moreover, why is it commented out!?